src/Http/Event/UpdateSubEventsRequestHandler.php

```diff
@@ -34,7 +31,8 @@ public function __construct(CommandBus $commandBus)
         $this->commandBus = $commandBus;
 
         $this->updateSubEventsParser = RequestBodyParserFactory::createBaseParser(
-            JsonSchemaValidatingRequestBodyParser::fromFile(JsonSchemaLocator::EVENT_SUB_EVENT_PATCH)
+            new JsonSchemaValidatingRequestBodyParser(JsonSchemaLocator::EVENT_SUB_EVENT_PATCH),
+            new DenormalizingRequestBodyParser(new SubEventUpdatesDenormalizer(), SubEventUpdates::class)
```

Comment:
If we extend this Symfony interface with one method `getClassName` we don't need the extra parameter  `SubEventUpdates::class` and can remove `\CultuurNet\UDB3\Event\ValueObjects\` 

So the `SubEventUpdatesDenormalizer` will need to implement one extra method, but that encapsulates the class name better.

Reply:
As agreed on Slack I've just aliased the Legacy VOs for now. I'll also mark more VOs as deprecated in another PR.